flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogGenericMetadataTest.java

```diff
@@ -65,6 +69,43 @@ public void testGenericTableSchema() throws Exception {
 		}
 	}
 
+	@Test
```

Comment:
Can add a comments: `NOTE: Do not modify this test, it is important to forward compatibility.`.